document_page/models/document_page.py

```diff
@@ -144,51 +151,9 @@ def write(self, vals):
     @api.model
     @api.returns('self', lambda value: value.id)
     def create(self, vals):
-        page_id = super(document_page, self).create(vals)
+        """Create the first history of a document."""
```

Comment:
S few lines below, shouldn't `self.create_history(...` rather be `result.create_history(...`